### ISSUE TITLE
SummaryJavadoc: need special processing of inheritDoc tag

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#3908]
taken from #3014

```
$ cat config.xml 
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="SummaryJavadoc">
        </module>
    </module>
</module>

$ cat Test.java 
public class Test extends Exception {

    /**
      * {@inheritDoc} */
    public String toString(){
        return "";
    }
}

$ java -jar checkstyle-7.6-all.jar -c config.xml Test.java 
Starting audit...
[ERROR] /home/rivanov/nfs/tests/Test.java:3: First sentence of Javadoc is incomplete (period is missing) or not present. [SummaryJavadoc]
Audit done.
Checkstyle ends with 1 errors.

```

Expected no error.

generated javadoc html is:
```
Method Detail

    toString

    public java.lang.String toString()

    Overrides:
        toString in class java.lang.Throwable


```

**Comment by ljacqu** [comment]
Extending this finding, it would be interesting to have a configuration option to skip this check for certain fragments. Specifically, I'm interested in enabling this check but for trivial getters I want to allow just having a concise `@return blabla` javadoc:

```java
/**
 * @return the category the article belongs to
 */
 public Category getCategory() {
   return category;
 }
```

**Comment by rnveach** [comment]
@ljacqu Create a new issue as this is only this one is for only dealing with `inheritDoc` tag.

**Comment by sagar-shah94** [comment]
I am on it

**Comment by romani** [comment]
@ljacqu , in new issue please start to define what is " trivial getters " . In my experience, setter/getter detection is hard and tricky, so definition of this term should be strict.

**Comment by romani** [comment]
fix is merged
